src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -225,16 +225,31 @@ def get_keyring_with_caps(self, entity: AuthEntity, caps: List[str]) -> str:
                 'entity': entity,
                 'caps': caps,
             })
-            if err:
+            if ret:
```

Comment:
A bit confused here. For `auth get-or-create` above we still examine `err` but for `auth caps` and `auth get` below we switched to `ret`. Is mixing these two ways intentional?

Reply:
nope, fixing!